backpack/core/derivatives/avgpoolnd.py

```diff
@@ -0,0 +1,179 @@
+"""The code relies on the insight that average pooling can be understood as
+convolution over single channels with a constant kernel."""
+
+import torch.nn
+from torch.nn import (
+    Conv1d,
+    Conv2d,
+    Conv3d,
+    ConvTranspose1d,
+    ConvTranspose2d,
+    ConvTranspose3d,
+)
+
+from backpack.core.derivatives.basederivatives import BaseDerivatives
+from backpack.utils.ein import eingroup
+
+
+class AvgPoolNDDerivatives(BaseDerivatives):
+    def __init__(self, N):
+        self.N = N
+        if self.N == 1:
+            self.conv = Conv1d
+            self.convt = ConvTranspose1d
+            self.equation = "v,n,c,l->vnc,l"
```

Comment:
Could you improve the name of that variable?

Reply:
of `self.equation`?